buildpacks/go/src/main.rs

```diff
@@ -139,11 +139,14 @@ impl Buildpack for GoBuildpack {
         }
         cmd::go_install(&packages, &go_env).map_err(GoBuildpackError::GoBuild)?;
 
-        log_header("Setting launch table");
-        let procs = proc::build_procs(&packages).map_err(GoBuildpackError::Proc)?;
-        log_info("Detected processes:");
-        for proc in &procs {
-            log_info(format!("  - {}: {}", proc.r#type, proc.command.join(" ")));
+        let mut procs: Vec<Process> = vec![];
```

Comment:
minor: this looks like this could be dropped and moved to line 145 without the `mut`

Reply:
`procs` is consumed further down (line 155 now), so it needs to be in this scope.